packages/peregrine/lib/store/actions/checkout/asyncActions.js

```diff
@@ -27,7 +27,6 @@ export const beginCheckout = () =>
             })
         );
         dispatch(getShippingMethods());
-        dispatch(getCountries());
```

Comment:
We no longer `getCountries` as part of the `beginCheckout` action. It happens in the `Flow` component now (where `beginCheckout` is called.